angular_components/CHANGELOG.md

```diff
@@ -1,3 +1,115 @@
+* Remove Sass mixin `button-text-capitalize()`.
+* Fix visibility in Microsoft Edge on Windows when using High Contrast mode.
+
+#### Material Datepicker
+* Add Sass mixins for margins around and between the next and previous buttons.
+* Increase the color ratio for the apply/cancel buttons.
+* Allow picking times using specified increments.
+* Handle `<ESC>` key press when picker is visible.
+* Removed deprecated `comparesToPreviousPeriod()` and
+  `comparesToSamePeriodLastYear()`.
+* Delegate to `FocusItemDirective`.
+* Fix the a11y role for the date range preset list.
+* Improved accessibility for presets.
+
+#### Material Dialog
+* Add Sass mixin for the footer margin.
+* Always keep a transparent scroll stroke to prevent dialog from shifting.
+
+#### Material Expansion Panel
+* Improve accessibility.
+* Accommodate contents taller than the screen size.
+
+#### Material Input
+* Make text size limits available to screen readers.
+* Change attribute `inputRole` to and input `role`.
+* Add a CSS class `.ltr` so input can set `direction: ltr` on the top section.
+* Add Sass mixin to adjust the location of floating label.
+
+#### Material Menu
+* Prevent screen readers from reading a non-collapsible label as "button".
+* Apply the same background color on focused and active items.
+* Update `icon` attribute to use a Dart boolean instead of a string `'true'`.
+* Add Sass mixin for adding ellipsis to overflowing item text.
+* Auto-activate FAB menu items when a screen reader triggers the fab.
+* Only auto focus an active item when it exists.
+* Protect against null pointer exception when closing.
+* Move focus-trap outside of the menu-item-groups.
+* Add a way to pass context to the `MenuItem` actions.
+
+#### Material Popup
+* Add `ariaLabel` input.
+* Encapsulate the popup class name.
+
+#### Material Select
+* Fix selected item visibility in Microsoft Edge on Windows when using High
+  Contrast mode.
+* Add Sass mixins for adding ellipsis to overflowing item text and a custom
+  outline.
+
+#### Material Slider
+* Support two sided sliders.
+
+#### Material Stepper
+* Add `yesText` input.
+* Add input to keep inactive steps in DOM.
+* Add partially complete state.
+* Update icon and index colors for improved a11y.
+
+#### Material Tab
+* Set tabbable tab to always be activeTab.
+* Improve `allow-text-wrap` Sass mixin.
+* Rename Sass mixin `allow-text-wrap` to `tab-text-wrap` and default
+  `$break-word` to `true`.
+* Add Sass mixin to apply text transform.
+
+#### Material Tooltip
+* Fix deferred content within tooltip.
+* Add Sass mixin for paper tool tips with multiple sections.
+
+#### Material Tree
+* Add input to toggle selection of non-leaf nodes.
+
+### Other Updates
+
+#### Miscellaneous
+* Remove `mat-icon-image()` in favor of `inline-image()`.
+* Add `subtract()` to Date model.
+* Migrate event handlers with multiple statements to component methods.
+* Fix violations of the `strict-raw-types` analysis option.
```

Comment:
`strict-raw-types` analysis option -> `--strict-raw-types` analyzer option?
(at first I was looking for "strict" in the linter options)

Reply:
Done.